cmd/earthly/registry_cmds.go

```diff
@@ -152,6 +153,53 @@ func (app *earthlyApp) actionRegistryLogin(cliCtx *cli.Context) error {
 		return fmt.Errorf("hosts is malformed")
 	}
 
+	switch app.registryCredHelper {
+	case "", "none":
+		return app.actionRegistrySetupUsernamePassword(cliCtx, path, cloudClient, host)
+	case "ecr-login":
+		return app.actionRegistrySetupECRLogin(cliCtx, path, cloudClient, host)
+	case "gcloud":
```

Comment:
might be nice to switch these to an enum and reuse them in the `actionRegistrySetupX` functions, and the other switch cases scenarios